src/offset/mod.rs

```diff
@@ -527,10 +527,55 @@ pub trait TimeZone: Sized + Clone {
     }
 }
 
+/// Helper function to map `LocalResult<FixedOffset>` to `LocalResult<DateTime<Local>>`.
+/// Returns `None` on out-of-range.
+fn local_time_min_offset<Tz: TimeZone>(
```

Comment:
Since this has a single caller, let's just inline it?

Reply:
With a separate function I could use `?`, and could convert to `LocalResult::None` in the caller. It seemed the cleanest solution at the time.

Found a nicer way to write it now.